terms.html

```diff
@@ -123,12 +130,22 @@
 verifiable data registry. For more information, see [[VC-DATA-MODEL]].
   </dd>
 
+  <dt><dfn>DID resolution</dfn></dt>
+
+  <dd>
+The function that takes as its input a <a>DID</a> and a set of resolution
+options and returns a <a>DID document</a> in a conforming representation
+plus additional metadata. This function relies on the "Read" operation of the
+applicable <a>DID method</a>. The contract of this function is defined
+in <a href="#did-resolution"></a>.
+  </dd>
+
   <dt><dfn data-lt="DID resolvers">DID resolver</dfn></dt>
 
   <dd>
-A system that is capable of retrieving a <a>DID document</a> for a given
-<a>DID</a>. These systems are specified in the DID Resolution specification
-[[DID-RESOLUTION]].
+A <a>DID resolver</a> is a software and/or hardware component that takes
+a <a>DID</a> as input and produces a <a>conforming DID document</a> as
+output.
```

Comment:
Suggest we define the component in terms of the function it implements.

```suggestion
A <a>DID resolver</a> is a software and/or hardware component that takes
a <a>DID</a> as input and produces a <a>conforming DID document</a> as
output by performing <a>DID resolution</a>.
```